packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/pubspec.yaml

```diff
@@ -51,7 +51,9 @@ dependencies: {{#enable_dev_menu}}
   widget_toolkit: ^0.2.0 {{#enable_pin_code}}
   widget_toolkit_biometrics: ^0.1.0 {{/enable_pin_code}} {{#enable_feature_otp}}
   widget_toolkit_otp: ^0.1.0{{/enable_feature_otp}}{{#enable_pin_code}}
-  widget_toolkit_pin: ^0.2.0{{/enable_pin_code}}
+  widget_toolkit_pin: ^0.2.0{{/enable_pin_code}}{{#enable_feature_qr_scanner}}
```

Comment:
Please make sure the latest version are used

Reply:
Done, set to 0.2.1